enterprise/backend/test/metabase_enterprise/sso/integrations/jwt_test.clj

```diff
@@ -191,24 +192,28 @@
                                                                          :for        "the new user"}
                                                                         default-jwt-secret))]
             (is (saml-test/successful-login? response))
-            (testing "new user"
-              (is (= [{:email        "newuser@metabase.com"
-                       :first_name   "New"
-                       :is_qbnewb    true
-                       :is_superuser false
-                       :id           true
-                       :last_name    "User"
-                       :date_joined  true
-                       :common_name  "New User"}]
-                     (->> (mt/boolean-ids-and-timestamps (t2/select User :email "newuser@metabase.com"))
-                          (map #(dissoc % :last_login))))))
-            (testing "User Invite Event is logged."
-              (is (= "newuser@metabase.com"
-                     (get-in (t2/select-one :model/AuditLog :topic :user-invited) [:details :email]))))
-            (testing "attributes"
-              (is (= {"more" "stuff"
-                      "for"  "the new user"}
-                     (t2/select-one-fn :login_attributes User :email "newuser@metabase.com"))))))))))
+            (let [new-user (t2/select-one User :email "newuser@metabase.com")]
+              (testing "new user"
+                (is (= {:email        "newuser@metabase.com"
+                        :first_name   "New"
+                        :is_qbnewb    true
+                        :is_superuser false
+                        :id           true
+                        :last_name    "User"
+                        :date_joined  true
+                        :common_name  "New User"}
+                       (-> (mt/boolean-ids-and-timestamps [new-user])
+                           first
+                           (dissoc :last_login)))))
+              (testing "User Invite Event is logged."
+                (is (= "newuser@metabase.com"
+                       (get-in (audit-log-test/latest-event :user-invited (:id new-user))
+                               [:details :email])
+                       #_(get-in (t2/select-one :model/AuditLog :topic :user-invited) [:details :email]))))
```

Comment:
Maybe committed this comment in by mistake?

Reply:
Yep thanks 😅 moving too quick